sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsOptions.cs

```diff
@@ -76,7 +76,8 @@ string IOptionsFormatter.Format()
         {
             JObject options = new JObject
             {
-                { nameof(MaxDegreeOfParallelism), MaxDegreeOfParallelism }
+                { nameof(MaxDegreeOfParallelism), MaxDegreeOfParallelism },
```

Comment:
I'm not sure about the previous change of adding MaxDequeueCount to this options type. It seems this was done to work around some other Azure Functions host.json config issue? As is, the approach seems partial, you're duplicating only one of the several other queue control knobs.

I'd like to understand the original issue better  - to see a repro of the actual issue.